app/views/docs/storage.phtml

```diff
@@ -181,9 +181,345 @@ File content.
     </li>
 </ul>
 
-<p>When you are trying to upload any files above 5MB, you will need to upload them in chunks. Don't worry; if you are using Appwrite's console or any of our SDKs, we handle that internally, so it will work no matter what file size you try to upload.
+<p>
+    When you are trying to upload any files above 5MB, you will need to upload them in chunks for better reliability and performance. If you're using an Appwrite SDK, this is handled automatically. If you're not using an SDK, you can learn more about file handling in the <a href="/docs/rest#file-handling">REST API</a> docs.
 </p>
 
+<h2><a href="/docs/storage#file-input" id="file-input">File Input</a></h2>
+
+<p>
+    Every language and platform handles file inputs differently. This section documents the expected input type of each SDK. Where applicable, Appwrite provides a <code>InputFile</code> class to accept multiple file sources, like paths, buffers, streams, or plain text.
+</p>
+
+<h3><a href="/docs/storage#file-input-client" id="file-input-client">Client SDK</a></h3>
+<ul class="phases clear" data-ui-phases>
+<li>
+    <h3>Web</h3>
+    <p>
+        The Appwrite Web SDK expects a <a href="https://developer.mozilla.org/en-US/docs/Web/API/File" target="_blank" rel="noopener">File</a> object for file creation. This is most commonly associated with DOM file inputs.
+    </p>
+
+    <p>
+        For example, for the input tag <code><?php echo htmlentities('<input type="file" id="uploader" multiple />'); ?></code>, you would call create file like this:
```

Comment:
```suggestion
        For example, for the input tag <code><?php echo htmlentities('<input type="file" id="uploader" />'); ?></code>, you would call create file like this:
```

I would remove `multiple`. That way browser only let's you pick one, so doing `[0]` later is cool.

If they want to upload multiple files, they would need for loop.